canonical/microk8s.io
# Apply /resources redesign

## Done

- Applired redesign as per [doc](https://docs.google.com/document/d/1PB5-CamGmSjhbuyuGSariC9zUW-8krAJD0l8JyvNk20/edit) and [design](https://www.figma.com/file/0DTIS3uNNUuDAdCKWrk8Uf/23.10---Microk8s.io?type=design&node-id=0-1&mode=design&t=IiSkurvj9PifdsoF-0)
**NOTE: This page is missing the suru intentionally. I created a reusable class for it in a [separate PR](https://github.com/canonical/microk8s.io/pull/610) which is being reviewed. I will add that class to this hero once approved**
## QA

- View the site locally in your web browser at: https://microk8s-io-607.demos.haus/resources
- Check page against doc and design 


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4192
